src/test/java/edu/cornell/kfs/sys/service/impl/CuUserProcurementProfileValidationServiceImplTest.java

```diff
@@ -16,7 +16,7 @@
 import edu.cornell.kfs.sys.businessobject.UserProcurementProfile;
 import edu.cornell.kfs.sys.service.UserProcurementProfileValidationService;
 
-@ConfigureContext(session = db18)
+@ConfigureContext(session = kad13)
```

Comment:
After a quick search of the base KFS unit test code, there could be a few things that could help with this. One is a UnitTestSqlDao helper object that can be used to run SQL on demand; I think SpringContext.getBean() can retrieve this if the test needs KFS running. Another thing that could help is the TestDataPreparator class, which can convert property file data into BOs or search criteria, and which can then be processed as needed.

Alternatively, you could modify or override the UserProcurementProfileValidationServiceImpl.canMaintainUserProcurementProfile method to work around the issue. However, depending on the workaround, that may defeat the purpose of the test.